packages/cli/src/controllers/users.controller.ts

```diff
@@ -41,6 +41,7 @@ import type {
 	UserRepository,
 } from '@db/repositories';
 
+@Authorized(['global', 'owner'])
```

Comment:
The `GET` operation in this controller must be public as all users should be able to list other users in order to display the sharing list.

Reply:
great catch. fixed, and added a test.